corehq/apps/settings/views.py

```diff
@@ -384,6 +387,14 @@ def dispatch(self, request, *args, **kwargs):
         # this is only here to add the login_required decorator
         return super(TwoFactorPhoneSetupView, self).dispatch(request, *args, **kwargs)
 
+    def done(self, form_list, **kwargs):
+        """
+        Store the device and reload the page.
+        """
+        self.get_device(user=self.request.user, name='backup').save()
+        messages.add_message(self.request, messages.SUCCESS, "Phone number added!")
+        return redirect(self.success_url)
```

Comment:
I think it would be better if you used the `reverse` function and referenced `TwoFactorProfileView.urlname` instead

Reply:
Done here:
https://github.com/dimagi/commcare-hq/pull/19574/commits/f98598c30fa6a865a4921562e98c04ac609f3623